prod-devel-rcar.yaml

```diff
@@ -175,6 +175,8 @@ components:
         - [PREFERRED_VERSION_xen, "4.17.0+git%"]
         - [PREFERRED_VERSION_xen-tools, "4.17.0+git%"]
 
+        - [PACKAGE_EXCLUDE:append, " qemu"]
```

Comment:
I am sure `qemu` is being installed because of some dependency. Should you fix the root cause of the problem? I.e. what `qemu` tries to install in the first place?

Reply:
I don't have an answer right now, I will investigate and come up later.

Reply:
As it turned out xen-tools recipe in meta-virtualization layer has a RRECOMMENDS containing qemu. This was added recently by the following commit: "80047c7 xen, xen-tools: add recommendation for Qemu for non-hvm x86" which is present in kirkstone branch. This is why we don't face the similar issue on dunfell (product's master branch so far). I will put a string to remove qemu from RRECOMMENDS into our corresponding bbappend.

Reply:
"+"